msal-dotnet-articles/acquiring-tokens/overview.md

```diff
@@ -14,65 +14,55 @@ ms.reviewer: localden
-
-Operating System | Library Platform | Kind of App | [Interactive Auth](./desktop-mobile/acquiring-tokens-interactively.md) | [IWA](./desktop-mobile/integrated-windows-authentication.md) | [U/P](./desktop-mobile/username-password-authentication.md) | [Device Code](./desktop-mobile/device-code-flow.md)
--- | -------- | --- | ----------  | --- | --- | ----------------- 
-<img alt="Windows Logo" src="/azure/active-directory/develop/media/index/logo_windows.svg" width="40" /> <br/> Windows desktop	| <img alt=".NET Logo" src="/azure/active-directory/develop/media/index/logo_net.svg" width="40" /> <br/> .NET| Desktop (WPF,<br/>  Windows.Forms,<br/>  Console) | Y | Y | Y | (Y)
-<img alt="Windows Logo" src="/azure/active-directory/develop/media/index/logo_windows.svg" width="40" /> <br/>Windows 10	| <img alt="Windows Logo" src="/azure/active-directory/develop/media/index/logo_windows.svg" width="40" /> <br/> UWP	| Store app	| Y	| Y	| 	
-<img alt="Android Logo" src="/azure/active-directory/develop/media/index/logo_android.svg" width="40" /> <br/> Android	| <img alt="Xamarin Logo" src="/azure/active-directory/develop/media/index/logo_xamarin.svg" width="40" /> <img alt="Android Logo" src="/azure/active-directory/develop/media/index/logo_android.svg" width="40" /> <br/> Xamarin Android	| Mobile	| Y	| 	| Y	
-<img alt="iOS Logo" src="/azure/active-directory/develop/media/index/logo_ios.svg" width="40" />| <img alt="Xamarin logo" src="/azure/active-directory/develop/media/index/logo_xamarin.svg" width="40" /> <img  alt="iOS Logo" src="/azure/active-directory/develop/media/index/logo_ios.svg" width="40" /><br/> Xamarin iOS	| Mobile	| Y	| 	| 	| Y
-Mac OS, Linux, Windows	| <img alt=".NET Core Logo" src="/azure/active-directory/develop/media/index/logo_netcore.svg" width="40" /> <br/> .NET Core 	| Console	| N/A see [1](/azure/active-directory/develop/msal-net-web-browsers)	| Y	| Y	| Y
+| Operating system      | Platform        | App type | [Interactive](./desktop-mobile/acquiring-tokens-interactively.md) | [IWA](./desktop-mobile/integrated-windows-authentication.md) | [ROPC](./desktop-mobile/username-password-authentication.md) | [Device Code](./desktop-mobile/device-code-flow.md) |
+|:----------------------|:----------------|:---------|:------------------------------------------------------------------|:-------------------------------------------------------------|:-------------------------------------------------------------|:----------------------------------------------------|
+| Windows (desktop)     | .NET            | Desktop (WPF, Windows Forms, Console) | ✅ | ✅ | ✅ | ✅ |
+| Windows 10 and newer  | UWP             | Store app                             | ✅ | ✅ | ❌ | ❌ |
+| Android               | Xamarin Android | Mobile                                | ✅ | ❌ | ✅ | ❌ |
+| iOS                   | Xamarin iOS     | Mobile                                | ✅ | ❌ | ❌ | ✅ |
```

Comment:
This matches the previous table, but wouldn't Device Code be available on all; and ROPC either available or not on both, Android and iOS? @bgavrilMS